LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Please use the MIT/expat license. We recently switched to it for all uplink libraries to avoid compatibility issues with GPLv2.
https://github.com/storj/uplink-c/blob/master/LICENSE

Reply:
Changed and squashed so it isn't in git history